cmc-ccd-adapter/build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+  id 'net.ltgt.apt' version '0.10'
+}
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  compile project(':cmc-domain-model')
+
+  compileOnly 'org.projectlombok:lombok:1.16.18'
```

Comment:
can you please be consistent with how you add dependencies,
first one uses single quotes
second double quotes
third is split out into group name and version,

preferred is to split it out into 3.

I realise you copied from the docs and they do the same thing :(.

Reply:
done